west.yml

```diff
@@ -68,7 +68,7 @@ manifest:
       groups:
         - hal
     - name: hal_gigadevice
-      revision: dd0e0322474462b58059e6fedaf1d67d2a0864d0
```

Comment:
`pull/29/head`

Reply:
Thanks! I made a screenshot of this, always forget how to do it;-).